client/packages.go

```diff
@@ -228,7 +231,8 @@ func (client *Client) FindOne(name string) (*Snap, *ResultInfo, error) {
 
 	snaps, ri, err := client.snapsFromPath("/v2/find", q)
```

Comment:
I would have expected a %w also inside snapsFromPath ?

Reply:
Nice catch, I updated the code and the tests now.